src/explicit/relative-pose.cc

```diff
@@ -149,8 +149,8 @@ namespace hpp {
         // convert p1 to Transform3f M1
         Transform3f M1 ((Quaternion_t (p1.vector ().tail <4> ()))
                         .toRotationMatrix (), p1.vector ().head <3> ());
-        // M2 = F_{2/J_2} M1 F_{2/J_2}
-        Transform3f M2 (frame2_ * M2 * frame2_.inverse ());
+        // M2 = F_{2/J_2} M1 F_{2/J_2}^{-1}
+        Transform3f M2 (frame2_ * M1 * frame2_.inverse ());
```

Comment:
Clearly, there was a typo in the previous code. I am not sure of the formula though. Did we wrote that formula somewhere ?

Reply:
Of course : https://github.com/humanoid-path-planner/hpp-constraints/blob/master/doc/right-hand-side.tex.